admin1.php

```diff
@@ -22,51 +22,89 @@
+			
+			<!-- login box -->
+			<div class = "login_box">
 				
-				<!-- login box -->
-				<div class = "login_box">
+				<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Add User</p></b>
 				
-					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw;">Add User</p></b>
-					
-					<h3><form method = "post" id= "01_login">
+				<!-- Form for input fields -->
+				<h3><form method = "post" id= "01_login">
 						
-						<p style = "margin: 2.3vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Username:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "text" name = "username" placeholder="Your Username..."/>
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 4vw 0vw 0vw 4.2vw;" type = "text" name = "username" placeholder="Username..."/>
 						
-						<p style = "margin: 1.5vw 0vw 0vw 3.7vw; font-size: 1.4vw;">Password:</p>
-						<input style = "width: 18.8vw; height: 1.9vw; margin: 1vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "password" name = "password" placeholder="Your Password..." id="myInput"/>
+					<input style = "width: 18.8vw; height: 1.9vw; margin: 3.05vw 0vw 0vw 4.2vw;" type = "password" name = "password" placeholder="Password..." id="myInput"/>
 						
-						<p style = "margin: 1.3vw 0vw -1.3vw 5.3vw; font-size: 1vw;">Show Password</p>
+					<p style = "margin: 1.4vw 0vw -1.2vw 5.7vw; font-size: 1vw;">Show Password</p>
 						
-						<!-- Checkbox for password visibility -->
-						<input type="checkbox" onclick="myFunction()"/>
+					<!-- Checkbox for password visibility -->
+					<input type="checkbox" onclick="myFunction()"/>
 						
-						<input style = "width: 19.2vw; height: 2.5vw; margin: 2.8vw 2vw 0vw 3.7vw; font-size: 1.2vw;" type = "submit" value = "Add User"/>
+					<input style = "width: 19.6vw; height: 2.5vw; margin: 4.4vw 0vw 0vw 4.2vw;" type = "submit" value = "Add User"/>
+					
+				</form></h3>
 					
-					</form></h3>
+				<center><p style = "margin: -6.5vw 0vw 0vw 0vw; font-size: 1vw; visibility: hidden;">Need An Account?<a href = "register.html"> Register</a></p></center>
 					
-					<center><p style = "margin: -6vw 0vw 0vw 0vw; font-size: 1vw; color: #ccb0f7; visibility: hidden;">Return Back To<a style = "color: #ccb0f7;" href = "LAYOUT_2 - login_v1.html"> Login</a></p></center>
+			</div>
 
+			<!-- content_1 to hold header, and content_2 -->
+			<div class = "content_1">
+			
+				<!-- header class which consists of the header text -->
+				<div class = "header">
+					<b><p style = "color: black; font-size: 2vw; margin: 1vw 0vw 0vw 1vw; ">Header</p></b>
 				</div>
-	
+
+				<!-- content_2, which contains content_3 -->
+				<div class = "content_2" style = "background-color: transparent; height: 17vw;">
+				
+					<!-- content_3, which contains the query row title and query row output via SQL -->
+					<div class = "content_3" style = "background-color: transparent;">

```

Comment:
Added content_3, which is contained inside contewnt_2 for display users query title rows and output.